<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>941</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Screenshot (404).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="loginsignup_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>881</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="login_b">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_b">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SIGN UP</string>
     </property>
    </widget>
    <widget class="QLabel" name="welcomemsg">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>431</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Pixeboy</family>
       <pointsize>30</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">YOUR EEG BASED AUTHENTICATION SYSTEM</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Block Stock</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MIND ID</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Block Stock</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Welcome to MIND ID</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="username_frame">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>371</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="line_usname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">ENTER USERNAME</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="usname">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="verify">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VERIFY</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="edf_frame">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>411</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="browse">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>BROWSE</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="file_validity">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>NO FILE  SELECTED</string>
     </property>
    </widget>
    <widget class="QLabel" name="line_edf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">SELECT .edf FILE</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SUBMIT</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="invalid_frame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>530</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LCD Solid</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string notr="true">Invalid User! Please enter a valid Username.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="precess_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="home_button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="QLabel" name="progress_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>271</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Block Stock</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">LOIGN         IN PROGRESS</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="email_frame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="email_line">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">ENTER YOUR EMAIL ID</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="verify_email">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LCD Solid</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VERIFY</string>
     </property>
    </widget>
   </widget>
   <zorder>label_2</zorder>
   <zorder>invalid_frame</zorder>
   <zorder>precess_frame</zorder>
   <zorder>username_frame</zorder>
   <zorder>edf_frame</zorder>
   <zorder>email_frame</zorder>
   <zorder>loginsignup_frame</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
